test/functional/DDR_Test/playlist.xml

```diff
@@ -57,8 +57,8 @@
 	-DTEST_RESROOT=$(TEST_RESROOT) -DRESOURCES_DIR=${RESOURCES_DIR} -DREPORTDIR=${REPORTDIR} -DOS=${OS} -DBITS=$(BITS) \
 	-Dtest.list=$(Q)TestDDRExtensionGeneral$(Q) -DADDITIONALEXPORTS=-showversion -f $(Q)$(TEST_RESROOT)$(D)tck_ddrext.xml$(Q); \
 	$(TEST_STATUS)</command>
-		<!-- temporarily disable this test on AIX, z/OS; github.com/eclipse/openj9/issues/1511 -->
-		<platformRequirements>^os.aix,^os.zos,^os.win</platformRequirements>
+		<!-- temporarily disable this test on z/OS; github.com/eclipse/openj9/issues/1511 -->
+		<platformRequirements>^os.zos,^os.win</platformRequirements>
```

Comment:
Why is this still disabled on Windows?

Reply:
There's a separate target for windows.

Reply:
Besides testDDRExt_General_win being part of the sanity set, and for both openj9 and ibm implementations, while testDDRExt_General_openj9 is extended and just openj9 impl, is there any difference between the targets?

Is it still required to be separate targets, now that all but z/os support is available?  @renfeiw, were there other reasons to keep separate targets?

Reply:
The ddr test on win is sanity and the ones on the other platforms are extended, this was the case before migration, so win and non-win are separated. testDDRExt_General_openj9 and testDDRExt_General_ibm separation is because we need to disable some platforms for openj9, these two can be combined when test can run on all platforms in openj9.